tests/integrationv2/Makefile

```diff
@@ -19,6 +19,8 @@ ifndef S2N_LIBCRYPTO
 	export S2N_LIBCRYPTO="openssl-1.1.1"
 endif
 
+LIBCRYPTO_ROOT:=/home/ubuntu/development/s2n/test-deps/openssl-1.1.1/
```

Comment:
This seems really fragile, no?

Reply:
It sure does. Shouldn't have gotten committed actually. I can remove it in my third PR.

Reply:
Updated here: https://github.com/awslabs/s2n/pull/1916